test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusIntegrationTestExtension.java

```diff
@@ -108,6 +109,11 @@ private IntegrationTestExtensionState ensureStarted(ExtensionContext extensionCo
                 state = doProcessStart(quarkusArtifactProperties, selectedProfile, extensionContext);
                 store.put(IntegrationTestExtensionState.class.getName(), state);
             } catch (Throwable e) {
+                Path appLogPath = PropertyTestUtil.getLogFilePath();
```

Comment:
An `IllegalStateException` is thrown if it can't determine the log file path. Wouldn't that mask the original exception?

Reply:
Ah, darn. I didn't look into the implementation. I'll fix it

Reply:
Fixed

Reply:
It would be easier if the method returned `null` or an `Optional<Path>` in this case but that may involve changing other places too. The current solution looks good enough 👍🏻